free-1.md

```diff
@@ -1,68 +1,198 @@
 # Free project 1
 
+Geometric Drawings
+
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+The user for this project is someone who wants to make geometric drawings.
+They might be a teacher, making drawings for handouts, slides, homework
+problems, or exam problems. They might be a researcher either geometric or in
+a related field trying to make a drawing to explain an idea, or perhaps making
+a drawing as a part of figuring out their ideas.
+
+Notably these people are careful and analytical, but they may not be
+computer-savvy.
 
+I want to build a language that allows them to make these drawings in a way
+that saves time and feels natural to them.
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
 
+The need is to create drawings, usually to be embedded in some sort of text
+document, but not always.
+
+Right now people do this in a few ways. The high school teachers I know use
+various plugins for Microsoft Word that have graphical interfaces. Others
+choose to use LaTeX and systems built on `tikz` like `tkz-euclid`, found
+[here](https://www.ctan.org/pkg/tkz-euclide?lang=en).
+
+The graphical interfaces are nice in some situations, but ambiguities emerge
+sometimes, because the only input to the program is (x,y) position, it may not
+always be clear what geometrically significant idea that location corresponds
+to. For example, if I draw a line from one angle of a triangle to the other
+side, am I drawing to the side's midpoint or bisecting the angle? These are
+usually not the same, but are close to each other!
+
+The tkz-euclid solution on the other hand, has a static textual interface. You
+can draw lines, draw shapes, and label intersections. The system is
+impressively complete, but seems cryptic at times. It is not clear how to do
```

Comment:
Agreed. I've never used tikz without spending most of my time looking up how to do something and fixing things I did wrong.